Tests/BowEffectsLaws/BracketLaws.swift

```diff
@@ -93,7 +94,17 @@ public class BracketLaws<F: Bracket & EquatableK> where F.E: Equatable & Arbitra
     private static func bracketMustRunReleaseTask() {
         property("bracketMustRunReleaseTask") <~ forAll { (a: Int, e: F.E) in
             var msg = 0
-            return F.pure(a).bracket(release: { i in msg = i; return F.pure(()) }, use: { _ -> Kind<F, Int> in throw e })
+            return F.pure(a).bracket(release: { i in F.pure(()).map { msg = i } }, use: { _ -> Kind<F, Int> in throw e })
```

Comment:
I think we could write it like: 
```suggestion
            return F.pure(a).bracket(release: { i in F.pure({ msg = i }) }, use: { _ -> Kind<F, Int> in throw e })
```

Reply:
Almost! I ended with `F.pure(msg = a)`